### PR TITLE
docs: Add empty lines in docstring [ci skip]

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Fixes docs by adding empty lines before sections such as `Args:` and `Return:`. Same as https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/284.

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py [hunk 1]
```diff
@@ -74,10 +74,12 @@ def train(self):
...
         """
         Entry point for ddp
+
         Args:
             process_idx:
             mp_queue: multiprocessing queue
             model:
+
         Returns:
         """
```
Comment (rohitgr7) [inline pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py:84]: mind add something here?

Comment (SkafteNicki) [reply pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py:84]: Function returns nothing, so it should probably just be removed

Comment (akihironitta) [reply pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py:84]: @rohitgr7 Thanks for your response! @SkafteNicki Agreed.

Comment (rohitgr7) [reply pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py]: I meant `process_idx` and `model` :]

**Review by Borda — APPROVED** [review]
lgtm

File: pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py [hunk 2]
```diff
@@ -110,9 +110,6 @@ def ddp_train(self, process_idx, model):
             process_idx:
             mp_queue: multiprocessing queue
             model:
-
-        Returns:
```
Comment (awaelchli) [inline pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py:114]: It just looks like these docstrings are outdated. It seems the method does in fact return a result.

Comment (Borda) [reply pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py:114]: @akihironitta mind revisit there you edited...

File: pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py (hunk 1 above)
Comment (akihironitta) [reply pytorch_lightning/accelerators/ddp_cpu_spawn_accelerator.py]: @rohitgr7 Thank you for the review and clarification! I am not familiar with distributed training, and thus I don't think I can update the docstring properly. Can you help if you know what to add to those variables?

File: pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py (hunk 2 above)
Comment (akihironitta) [reply pytorch_lightning/accelerators/ddp_torchelastic_accelerator.py:114]: @awaelchli Thank you for your review! You're right. It actually returns `results`, but I have no idea what explanation to add. Can you maybe help?

**Comment by akihironitta** [comment]
Might be off the topic, but it seems a bit strange that the method `ddp_train` of `DDP2Accelerator` returns `results` while `ddp_train` of `DDPSpawnAccelerator` returns nothing (though I am not familiar with distributed training). Just curious if it's on purpose.
https://github.com/PyTorchLightning/pytorch-lightning/blob/eddf35a379efbd1b8fbedddb0d4b550e8e5ca48d/pytorch_lightning/accelerators/ddp_spawn_accelerator.py#L76
https://github.com/PyTorchLightning/pytorch-lightning/blob/f37444fa3e82011e4a71b5ca8dc897eff9ba0fa3/pytorch_lightning/accelerators/ddp2_accelerator.py#L120

**Review by tchaton — APPROVED** [review]
Great PR ! Thanks for your contributions to Pytorch Lightning !

**Comment by tchaton** [comment]
Hey @akihironitta, would you mind creating an issue concerning `ddp_train` ?

File: pytorch_lightning/accelerators/ddp2_accelerator.py [hunk 3]
```diff
@@ -123,9 +123,6 @@ def ddp_train(self, process_idx, mp_queue, model):
             process_idx:
             mp_queue: multiprocessing queue
             model:
```
Comment (rohitgr7) [inline pytorch_lightning/accelerators/ddp2_accelerator.py:125]: @akihironitta update this everywhere?
```suggestion
            process_idx: current process rank
            mp_queue: multiprocessing queue
            model: pointer to current :class:`LightningModule`
```

**Review by justusschock — APPROVED** [review]

File: pytorch_lightning/accelerators/ddp2_accelerator.py (hunk 3 above)
Comment (akihironitta) [reply pytorch_lightning/accelerators/ddp2_accelerator.py:125]: @rohitgr7 Sure. Thank you very much for your help!

**Pull request merged**